doc/features/currencies.rst

```diff
@@ -49,6 +49,36 @@ The Bitcoin currencies provides a single currency: the Bitcoin. It uses XBT as i
     $currencies->contains(new Currency('USD')); // returns boolean whether USD is available in this repository (false)
     $currencies->subunitFor(new Currency('XBT')); // returns the subunit for the Bitcoin (8)
 
+CurrencyList
+-----------------
```

Comment:
What do you mean? Does it need to be the same lenth as CurrencyList So it means 13 -?